packages/http/libraries/fastify/src/adapter/InversifyFastifyHttpAdapter.ts

```diff
@@ -24,7 +25,7 @@ import { Container } from 'inversify';
 export class InversifyFastifyHttpAdapter extends InversifyHttpAdapter<
   FastifyRequest,
   FastifyReply,
-  (err?: unknown) => void,
+  HookHandlerDoneFunction,
```

Comment:
We are allowing to throw non Error values, I would prefer to keep it that way.

I see this adapter can be updated to throw non Error values same way the other adapters are implemented